cargo-shuttle/src/client.rs

```diff
@@ -182,12 +182,22 @@ impl Client {
         self.get(path).await
     }
 
-    pub async fn delete_project(&self, project: &ProjectName) -> Result<project::Response> {
+    pub async fn stop_project(&self, project: &ProjectName) -> Result<project::Response> {
         let path = format!("/projects/{}", project.as_str());
 
         self.delete(path).await
     }
 
+    pub async fn delete_project(&self, project: &ProjectName, dry_run: bool) -> Result<String> {
```

Comment:
**suggestion**: use an enum instead of a bool for the dry run. `delete_projects("foo", DryRun::True)` is clearer that `delete_projects("foo", true)`.

Reply:
Too verbose for my taste, considering it would bloat the codebase if applied everywhere.

Reply:
My point is not to do it everywhere, just here, since it has quite the impact on the operation.

Reply:
I'll skip it since it is only being used in one place.